Input/Input.module.less

```diff
@@ -121,6 +121,10 @@
 
 		.keypad {
 			margin: @sand-input-fullscreen-keypad-margin;
+
+			&.mobile {
```

Comment:
I suggest using `max-width` rather than adding this class.
We could confine the width of the keypad to viewport width by
```
max-width: 100vw;
```

Let's avoid using "platform distinguish strategy" and it would be better if you'd think this is more like making it responsive.